app/assets/stylesheets/application.scss.erb

```diff
@@ -3464,6 +3464,16 @@ color: var(--black);
     }
   }
 
+  a.gap {
```

Comment:
This is to prevent the '...' in the pagination interface from being highlighted on mouseover